docs/source/quickstart.rst

```diff
@@ -0,0 +1,80 @@
+Quickstart
+==========
+
+Installation
+------------
+
+If you are using `Anaconda`_ or Miniconda::
+
+    conda install -c conda-forge xrviz
+
+If you are using virtualenv/pip::
+
+    pip install xrviz
+
+Note that this will install with the mininum of optional requirements.
+If you want a more complete install, use conda instead.
+
+If you want to develop::
+
+    git clone https://github.com/intake/xrviz
+    cd xrviz
+    python setup.py develop
+
```

Comment:
This is what I do to develop. Please let me know if there is a better alternative command that should be mentioned here.

Reply:
Prefer
```
git clone https://github.com/intake/xrviz
cd xrviz
pip install -e .
```
or
```
git clone https://github.com/intake/xrviz
pip install -e xrviz
```

Reply:
Thanks!